app/views/planning_applications/assessment/conditions/_form.html.erb

```diff
@@ -8,19 +8,15 @@
   <%= content_tag :div, class: "govuk-!-margin-bottom-5" do %>
     <%= form.hidden_field :id %>
     <%= form.hidden_field :standard, value: false %>
```

Comment:
Are these hidden fields needed?

Reply:
We need to update `standard` but actually this could be done in the controller.